worker/dbaccessor/worker_test.go

```diff
@@ -249,6 +257,43 @@ func (s *workerSuite) TestWorkerStartupExistingNode(c *gc.C) {
 	workertest.CleanKill(c, w)
 }
 
+func (s *workerSuite) TestWorkerStartupExistingNodeWithLoopbackPreferred(c *gc.C) {
+	defer s.setupMocks(c).Finish()
+
+	s.expectAnyLogs()
+	s.expectClock()
+	s.expectTrackedDBKill()
+
+	mgrExp := s.nodeManager.EXPECT()
+	mgrExp.EnsureDataDir().Return(c.MkDir(), nil)
+
+	// If this is an existing node, we do not invoke the address or cluster
+	// options, but if the node is not as bootstrapped, we do assume it is
+	// part of a cluster, and does not use the TLS option.
+	// IsBootstrapped node is called twice - once to check the startup
+	// conditions and then again upon worker shutdown.
+	mgrExp.IsExistingNode().Return(true, nil)
+	mgrExp.IsLoopbackBound(gomock.Any()).Return(true, nil).Times(2)
+	mgrExp.IsLoopbackPreferred().Return(false)
```

Comment:
Should this be true?

Reply:
Yes. The comments need updating too (my fault) `IsBootstrappedNode` was removed and replaced with `IsLoopbackBound`.